src/room_member.cpp

```diff
@@ -175,8 +175,7 @@ void RoomMember::ReceiveLoop() {
 };
 
 void RoomMember::Join(const std::string& nickname, const std::string& server, uint16_t server_port, uint16_t client_port) {
-    // TODO(Subv): Use client_port.
-    RakNet::SocketDescriptor socket;
+    RakNet::SocketDescriptor socket(client_port, 0);
```

Comment:
Thats right, but RakNet tries to set the value 0 to the memory area the nullptr is pointing to. Which will fail with a BAD_ACCESS error...